<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QIseCaliBrateResult</class>
 <widget class="BaseDlg" name="QIseCaliBrateResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCaliBrateResult</string>
  </property>
  <widget class="QWidget" name="bk_widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1115</width>
     <height>120</height>
    </rect>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">all</string>
   </property>
   <property name="bks" stdset="0">
    <string notr="true">gray</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>235</y>
     <width>621</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>当前结果</string>
   </property>
  </widget>
  <widget class="QTableView" name="curve_cur">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>287</y>
     <width>1115</width>
     <height>180</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>44</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="curve_his">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>548</y>
     <width>1115</width>
     <height>180</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>44</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>1081</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label0">
        <property name="text">
         <string>项目名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>校准时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>单位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>校准品批号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>IC批号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>BS批号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>287</y>
     <width>1115</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>287</y>
     <width>1</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>466</y>
     <width>1115</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>727</y>
     <width>1115</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>548</y>
     <width>1115</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1164</x>
     <y>287</y>
     <width>1</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>548</y>
     <width>1</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_4">
   <property name="geometry">
    <rect>
     <x>1164</x>
     <y>548</y>
     <width>1</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>750</y>
     <width>1121</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="update_btn">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_none_flat_btn">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="print_none_flat_btn">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>打印</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="close_btn">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>495</y>
     <width>1115</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>历史结果</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>校准时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="curve_his_times">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
